frontend/src/App.js

```diff
@@ -1,25 +1,38 @@
 import logo from './logo.svg';
 import './App.css';
+import React from "react";
+import { Routes, Route, Link } from "react-router-dom"
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+      <div className="App">
+        <Routes>
```

Comment:
Let's have these routes in `index.js`. In the top most route let's have an `index` route which will have a `Layout` component with an `Outlet` (see what an [`Outlet`](https://reactrouter.com/en/main/components/outlet) is and how to use it ). This is so that components like `Navbar` and `PageFooter` can be rendered on every page